pxtlib/webusb.ts

```diff
@@ -197,40 +225,38 @@ namespace pxt.usb {
 
             this.enabled = true;
             this.log("registering webusb events");
-            (navigator as any).usb.addEventListener('disconnect', this.handleUSBDisconnected, false);
-            (navigator as any).usb.addEventListener('connect', this.handleUSBConnected, false);
+            navigator.usb?.addEventListener('disconnect', this.handleUSBDisconnected, false);
```

Comment:
Similar to Eric's questions about safe continuation. Does this need 
```suggestion
            navigator?.usb?.addEventListener('disconnect', this.handleUSBDisconnected, false);
```
like there are in other places or is just having `.usb` fine in this case? Same question applies to the line below.

Reply:
https://github.com/microsoft/pxt/blob/dev/jwunderl/async-ify-webusb-cleanup/pxtlib/webusb.ts#L608 is where we check for it being enabled for the `pxt.usb.isEnabled()` which we use to gate all other calls; but adding in for clarity sake~